apps/fabric-website/src/pages/Styles/ThemeSlotsPage/docs/web/ThemeSlotsImplementation.md

```diff
@@ -0,0 +1,10 @@
+Colors can be applied to text, backgrounds, or borders using the following class name conventions:
```

Comment:
nit: Feels like this implementation section needs a revamp of some sort. Currently it only shows you the Fabric Core application. Should we also include a css-in-js story? Or I missed it and we have it somewhere else on the website?

Reply:
Agreed with @Vitalius1 's comments, this page could probably connect with the [Theming wiki page](https://github.com/OfficeDev/office-ui-fabric-react/wiki/Theming). We should probably discuss if it'd make sense to move some of that content here or keep it separate. I don't think we should block this PR on that, but would be curious to hear @phkuo and @ecraig12345 's thoughts.

Reply:
@jordandrako ok, these improvements look great!